modules/gatk4/intervallisttools/main.nf

```diff
@@ -51,4 +51,15 @@ process GATK4_INTERVALLISTTOOLS {
         gatk4: \$(echo \$(gatk --version 2>&1) | sed 's/^.*(GATK) v//; s/ .*\$//')
     END_VERSIONS
     """
+
+    stub:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    mkdir ${prefix}_split
```

Comment:
Have you checked that just creating the folder works, the output channel is declared in this manner:
```
 tuple val(meta), path("*_split/*/*.interval_list"), emit: interval_list
 ```
 so I would say that should maintain the same structure

Reply:
I agree! I have fixed it now.

Reply:
🚀